tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Modified version of script from here:
+#   http://stackoverflow.com/questions/26082444/how-to-work-around-travis-cis-4mb-output-limit
```

Comment:
It's unfortunate that we need a wrapper script here. Could we just suppress the gradle output instead? E.g.:

```groovy
  testLogging {
    showStandardStreams = false
  }
```

We could always set this back to ```true``` when we need to debug locally.